python/ray/tests/test_basic.py

```diff
@@ -1651,6 +1655,45 @@ def use_actor_handle(handle):
 
     ray.get(use_actor_handle.remote(test_actor))
 
+    # Check that exceptions are deferred until ray.get().
+
+    exception_str = 'test_basic remote task exception'
```

Comment:
```suggestion
    exception_str = "test_basic remote task exception"
```

This is arguably too nit picky but is enforced by the linter.